src/data.jl

```diff
@@ -286,8 +287,11 @@ raMODFTData(x::raMOInput) = x.dftdata
 Electrum.Crystal(x::raMOInput) = x.dftdata.xtal
 Electrum.basis(x::raMOInput) = basis(x.dftdata.xtal)
 Electrum.PlanewaveWavefunction(x::raMOInput) = x.dftdata.wave
+Electrum.PeriodicAtomList(x::raMOInput) = x.dftdata.xtal.atoms
+Electrum.supercell(x::raMOInput) = supercell(x.dftdata)
 # Index by run
 Base.getindex(x::raMOInput, i) = x.runlist[i]
+Base.iterate(x::raMOInput, i=1) = i > length(x.runlist) ? nothing : (x.runlist[i], i+1)
```

Comment:
In general, you want to define `Base.size` and `Base.getindex` for `AbstractArray` subtypes, which gives you a default array iterator. However, considering the complexity of `raMOInput`, I'll leave this design decision up to you: it might not make much sense to define a `size` when it contains multiple objects that have their own size.

Reply:
I think defining `Base.size` as the length of the `runlist` will be appropriate, since other items with variable sizes will be in the `dftdata` substruct.

Reply:
Awesome, just define `Base.size` using the size of `runlist` and this should be good to go.

Reply:
Resolved in [bd5c3eb](https://github.com/xamberl/DFT-raMO/pull/39/commits/bd5c3eb8cfd93d5212009ac634f7a436c6a16b6d)